src/awsconfig/client.go

```diff
@@ -66,21 +87,41 @@ func (a *awsLoader) Import(data []byte) error {
 
 // Initialize
 func (a *awsLoader) Initialize() error {
-	// pull all the config down for this service
+	// get the env config for the service
 	prefix := "/" + a.environment + "/" + a.serviceName + "/"
-	globalPrefix := "/" + a.environment + "/global/"
-	var err error
 
-	a.config, err = a.pullConfigWithPrefix(globalPrefix, nil) // pull global config
+	serviceConfig, err := a.pullConfigWithPrefix(prefix, nil) // pull service specific config
 	if err != nil {
 		return err
 	}
 
-	serviceConfig, err := a.pullConfigWithPrefix(prefix, nil) // pull service specific config
+	// if the envoronment is local then we want to
+	// get the config of the $CONFIG_USER as well and merge
+	// that with the service config from above
+	if a.environment == "local" {
+		// get the user config
+		prefix = "/" + os.Getenv("CONFIG_USER") + "/" + a.serviceName + "/"
```

Comment:
Using an env var here feels weird.  What happens if its not set?  

Do we need a 3rd level of overriding?  I was thinking /kyle would just be is own separate environment.  If we want it as an override it *might* make more sense to treat it more like `/local/global`. i.e. `/local/context/kyle/<serviceName>/...`

Reply:
I'd like to be consistent with the first section being an environment: `/<env>/` i think it gets weird if sometimes its a user specific override instead.

Reply:
I don't think we want to have `/kyle` be its own separate environment because then each person will have to maintain their own `/kyle/global` which I would rather not have to do. If we make some changes then everyone has to do it as well. I think that it is still best to have 3 levels of overriding so you can just use everything in `/local/global` and `local/servive_name` still. I chose to use `CONFIG_USER` because then I don't have to check 
```
if environment != "dev" || environment != "stg" || environment != "prod"
```
Then there is no problem here if we decide to add other environments or something as well

Reply:
Also I made `CONFIG_USER` part of the docker-compose file in the service set the $USER so it should always get set but worth adding a check